BlinkID/android/src/main/java/com/microblink/flutter/MicroblinkCreationParams.java

```diff
@@ -0,0 +1,17 @@
+package com.microblink.flutter;
+
+
+import java.util.Map;
+
+public class MicroblinkCreationParams {
```

Comment:
I don't think the introduction of this class brings a significant value. Besides, I'd argue about its name.